smart-contracts/test/Lock/erc721/getApproved.js

```diff
@@ -24,15 +24,15 @@ contract('Lock ERC721', (accounts) => {
       })
     })
 
-    it('should return the address of the approved owner for a key', () => {
-      return locks['FIRST'].approve(accounts[3], accounts[3], {
-        from: lockOwner
-      }).then(() => {
-        return locks['FIRST'].getApproved(accounts[3])
-      }).then((approved) => {
-        assert.equal(accounts[3], approved)
-      })
-    })
+    // it('should return the address of the approved owner for a key', () => {
```

Comment:
could you use `it.skip` rather than commenting out the whole test? This means that it will stay on our radar more easily ;)

Reply:
Just seeing this now. Sorry I missed this one. Will get it fixed up first thing in the morning and re-submit.

Reply:
Awesome! Thanks!